Templates/header.html.twig

```diff
@@ -12,7 +12,7 @@
                 <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded text-white" href="http://localhost:8888/P5_Blog_Guillaume_De_Backre/#about">CV</a></li>
                 <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded text-white" href="http://localhost:8888/P5_Blog_Guillaume_De_Backre/#contact">Contact</a></li>
                 <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded text-white" href="?action=listPosts">Blog</a></li>
-                <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded text-white" href="?action=listPosts">Connexion</a></li>
+                <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded text-white" href="http://localhost:8888/P5_Blog_Guillaume_De_Backre/inscription">Inscription</a></li>
```

Comment:
You should put relative path instead of absolute path.

```html
<li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded text-white" href="/inscription">Inscription</a></li>

```